zappa/handler.py

```diff
@@ -321,6 +321,17 @@ def get_function_for_aws_event(self, record):
 
         return None
 
+    # AWS MSK
+    def get_function_for_kafka_trigger(self, event):
+        arn = event.get("eventSourceArn")
+        # if multiple topics are in 1 trigger, need to improve
+        keys = list(event.get('records').keys())
+        topic = keys[0].rsplit('-', 1)[0]
+        arn = '{}:{}'.format(arn, topic.strip())
+        if arn:
```

Comment:
Since arn is '{}:{}' it will always contain ':' and always evaluate to `True`.
This `if` should probably be above, `arn = '{}:{}'.format(arn, topic.strip())`.

Reply:
btw - it's ok to use f-strings now. (initial code still supported 2.6)